mautic/mautic
# Update bots list

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Based on BotTracker https://github.com/Thomas--F/BotTracker/blob/master/botlist.txt for Matomo analytics I update bots list

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. See If bots list in configuration is much large

Code review is enough
